### PR TITLE
[typescript-sdk] Add documentation for typescript sdk

### PR DESCRIPTION
**Pull request by vchs1v**
## Motivation

This PR adds a documentation for Aptos typescript SDK in order to clarify its functionality

### Have you read the [Contributing Guidelines on pull requests](https://github.com/aptos-labs/aptos-core/blob/main/CONTRIBUTING.md#pull-requests)?

Yes

## Test Plan

There is no need for test

### CODE REVIEW

**Comment by davidiw** [comment]
Nice PR, will leave it up to the main devs :)

**Comment by jjleng** [comment]
Really appreciate your PR. Here are some thoughts:
1) A lot of the class/API descriptions do not add much value. For example, it will great if you can provide more details, such as what a `accountAddress` looks like.
2) It is better to write good code comments and generate the class/API docs with something like TSDoc. 

BTW, we are working on adding more code comments that are friendly to TSDoc. You could expect to see more SDK docs soon.

**Comment by vchs1v** [comment]
@jjleng I have documented all methods with TSDoc. If you see some ways to improve it, lets discuss

File: ecosystem/typescript/sdk/src/token_client.ts
Comment (CapCap) [inline ecosystem/typescript/sdk/src/token_client.ts:269]: is this correct? `in next form`

Comment (vchs1v) [reply ecosystem/typescript/sdk/src/token_client.ts:269]: I tried to pay attention to a Token object interface, that is below.

Comment (jjleng) [reply ecosystem/typescript/sdk/src/token_client.ts:269]: `@returns Token object in below format` ?

File: ecosystem/typescript/sdk/src/token_client.ts
Comment (jjleng) [inline ecosystem/typescript/sdk/src/token_client.ts:224]: `in below format`

File: ecosystem/typescript/sdk/src/token_client.ts
Comment (jjleng) [inline ecosystem/typescript/sdk/src/token_client.ts:189]: `in below format`

File: ecosystem/typescript/sdk/src/aptos_client.ts
Comment (jjleng) [inline ecosystem/typescript/sdk/src/aptos_client.ts:354]: This is not accurate. Lets remove **It is a hex-encoded string prefixed with `0x` and fulfilled with two hex digits per byte**. The bytes are just BCS bytes, no hex-encoding. We need to update the openapi spec as well. I will do that.

**Review by jjleng — APPROVED** [review]
looks good. approved 🚀

The TSDoc for `submitSignedBCSTransaction` is inaccurate. Please fix before merging.

File: ecosystem/typescript/sdk/src/aptos_account.ts
Comment (jjleng) [inline ecosystem/typescript/sdk/src/aptos_account.ts:113]: // ts-ignore?

Comment (vchs1v) [reply ecosystem/typescript/sdk/src/aptos_account.ts]: Seems like eslint does not recognize `eslint-disable` within TSDoc comments. I decided to just use template string literal

**Pull request closed without merging**
